### DISCUSSION TITLE
Setting ctf to ang

### DISCUSSION THREAD

**Discussion by ZephanDCruz** [mtex-toolbox/mtex#1589] [Ask Anything] [answered]
Hello All,

I would like to convert my ctf files to ang files.
mtex has a bit of matlab code that allows me to this, however
I am not terribly familiar with matlab and do not know how to input my files!

What should I be typing in the arguments for the function?
What should be entered for the "@EBSD" part and "varargin?"

Thank you!

**Comment by ZephanDCruz** [comment]
I have poked around and have figure out that I had to reference my ebsd workspace item that I made with the ebsd.load function.
However, now I get an error: shown here

![image](https://user-images.githubusercontent.com/115608386/195256151-e8665321-a5d4-4eb8-9830-d898e64a474e.png)

Does this means my data is "bad?"
How do I fix this issue?

**Reply by ralfHielscher** [reply]
Can you type ```ebsd``` into the command window and poste the output?

Ralf.

**Reply by ZephanDCruz** [reply]
I slept on it, and retried the commands and everything worked!
I'm not entirely sure what I did differently, but I've saved my file so I'm fixed.
Thank you!

**Comment by ralfHielscher** [comment, marked as answer]
For exporting something you first need to import the data, i.e.
```matlab
ebsd = EBSD.load('myFile.ctf')
export_ang(ebsd,'myFile.ang')
```
Ralf.

**Reply by ZephanDCruz** [reply]
Thank you for your quick response Ralf!
Importing the data works, but I am still not able to export data.
Mtex gives an error:
`>> export_ang(@ebsd, 'slice 1 1.ang')

------------------------------------------------------
     Exporting 'ang' file 
------------------------------------------------------
 -> Collecting data
Dot indexing is not supported for variables of this type.

Error in export_ang (line 32)
ebsd.phaseMap = ebsd.phaseMap - (min(ebsd.phaseMap)+1); %Adapt *.ang phase map convention`

Why could this be happening?
How can I fix this issue?
Thank you!
Zephan
